src/MainWindow.cpp

```diff
@@ -2790,18 +2808,18 @@ void MainWindow::onUseLimitedFontsPreferenceChanged(bool flag)
     }
 }
 
-void MainWindow::onShowNoteThumbnailsPreferenceChanged(bool flag)
+void MainWindow::onShowNoteThumbnailsPreferenceChanged()
 {
-    QNDEBUG(QStringLiteral("MainWindow::onShowNoteThumbnailsPreferenceChanged: ")
-            << (flag ? QStringLiteral("enabled") : QStringLiteral("disabled")));
+    QNDEBUG(QStringLiteral("MainWindow::onShowNoteThumbnailsPreferenceChanged"));
```

Comment:
It's generally better to leave an empty line after `QNDEBUG` with method name, unless what follows are just one-two lines.